### PR TITLE
Create 2021-72.md

### PR DESCRIPTION
**Pull request by billfox1**

### CODE REVIEW

**Review by philidem — APPROVED** [review]

File: release-notes/2021-72.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+2021-06-02
+
+## New Features and Improvements
+
+- Improved tooltip placement on the gear menu so that the tooltips no longer cover up other menu items
+
+- Newly imported CIS Benchmarks will now have Policies & Procedures disabled by default
+
+- Billing & products page has been redesigned so it is easier to use now
```
Comment (erkangz) [inline release-notes/2021-72.md:11]: By "Billing", do you mean the "In-app purchase" page?

File: release-notes/2021-72.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+- When filtering findings & vulnerabilities, filtering by `informational` will also include `informative` and `info`
+
+- When filtering findings & vulnerabilities, filtering by `no value` will also include `undefined` and `null`
```
Comment (erkangz) [inline release-notes/2021-72.md:15]: Can this be combined with the previous bullet?

File: release-notes/2021-72.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  `principal` properties on IAM permission relationships
+
+- Added `normalizedActions` property on IAM permission relationships to store
+  action values in all lowercase
+
+- Added support for ingesting the following **new** resources:
+
+  | Service | Resource / Entity    |
+  | ------- | -------------------- |
+  | WAF V2  | `aws_waf_v2_web_acl` |
```
Comment (erkangz) [inline release-notes/2021-72.md:32]: `WAF` not `WAF V2` any more. 

We renamed WAF V1 to `WAF Classic`

File: release-notes/2021-72.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+- Billing & products page has been redesigned so it is easier to use now
+
+- When filtering findings & vulnerabilities, filtering by `informational` will also include `informative` and `info`
+
+- When filtering findings & vulnerabilities, filtering by `no value` will also include `undefined` and `null`
+
+## Integrations
+
+### AWS
+
```
Comment (erkangz) [inline release-notes/2021-72.md:20]: We should add VPC Endpoints and endpoint policies

File: release-notes/2021-72.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+  | `azure_resource_group`   | `HAS`      | `azure_gallery`           |
+  | `azure_gallery`          | `CONTAINS` | `azure_shared_image`      |
+  | `azure_vm`               | `HAS`      | `azure_shared_image`      |
+  | `azure_vm`               | `HAS`      | `azure_image`             |
+  | `azure_management_group` | `CONTAINS` | `azure_management_group`  |
+  | `azure_account`          | `HAS`      | `azure_management_group`  |
+  | `azure_vm`               | `USES`     | `azure_service_principal` |
+
+
+
```
Comment (erkangz) [inline release-notes/2021-72.md:85]: delete the extra spaces/empty lines

File: release-notes/2021-72.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+  | Source                           | \_class | Target                  |
+  | -------------------------------- | ------- | ----------------------- |
+  | `google_bigquery_dataset`        | `HAS`   | `google_bigquery_table` |
+  | `google_bigquery_dataset`        | `USES`  | `google_kms_crypto_key` |
+  | `google_compute_disk`            | `USES`  | `google_kms_crypto_key` |
+  | `google_sql_mysql_instance`      | `USES`  | `google_kms_crypto_key` |
+  | `google_sql_postgres_instance`   | `USES`  | `google_kms_crypto_key` |
+  | `google_sql_sql_server_instance` | `USES`  | `google_kms_crypto_key` |
+
+
```
Comment (erkangz) [inline release-notes/2021-72.md:112]: extra line here

File: release-notes/2021-72.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+  | `google_compute_disk`            | `kmsKeyServiceAccount`, `kmsKeyName`                                                       |
+  | `google_compute_instance`        | `integrityMonitoringEnabled`, `secureBootEnabled`, `vtpmEnabled`, `connectedNetworksCount` |
+  | `google_sql_mysql_instance`      | `kmsKeyName`                                                                               |
+  | `google_sql_postgres_instance`   | `kmsKeyName`                                                                               |
+  | `google_sql_sql_server_instance` | `kmsKeyName`                                                                               |
+  | `google_storage_bucket`          | `kmsKeyName`                                                                               |
+  | `google_bigquery_dataset`        | `_class: Database` [#175](https://github.com/JupiterOne/graph-google-cloud/issues/175)     |
+  | `All Entities`                   | `projectId`                                                                                |
+
+
```
Comment (erkangz) [inline release-notes/2021-72.md:126]: extra line

**Review by erkangz — CHANGES_REQUESTED** [review]

File: release-notes/2021-72.md (hunk 3 above)
Comment (billfox1) [reply release-notes/2021-72.md:32]: In Adam's absence should I just remove "V2"?  And what about "`aws_waf_v2_web_acl`"?

File: release-notes/2021-72.md (hunk 1 above)
Comment (billfox1) [reply release-notes/2021-72.md:11]: Deferring to Carter.

File: release-notes/2021-72.md (hunk 2 above)
Comment (billfox1) [reply release-notes/2021-72.md:15]: Deferring to Carter.

File: release-notes/2021-72.md (hunk 4 above)
Comment (billfox1) [reply release-notes/2021-72.md:20]: In Adam's absence I'll need guidance here.

File: release-notes/2021-72.md (hunk 1 above)
Comment (hesto2) [reply release-notes/2021-72.md:11]: Yes

File: release-notes/2021-72.md (hunk 2 above)
Comment (hesto2) [reply release-notes/2021-72.md:15]: Yep

File: release-notes/2021-72.md (hunk 4 above)
Comment (billfox1) [reply release-notes/2021-72.md:20]: Updated with Kenan's assistance

File: release-notes/2021-72.md (hunk 3 above)
Comment (billfox1) [reply release-notes/2021-72.md]: Updated with Kenan's assistance

File: release-notes/2021-72.md (hunk 4 above)
Comment (billfox1) [reply release-notes/2021-72.md:20]: Updated with Kenan's assistance

File: release-notes/2021-72.md (hunk 3 above)
Comment (billfox1) [reply release-notes/2021-72.md]: Updated with Kenan's assistance

**Review by erkangz — APPROVED** [review]

**Pull request merged**